perf/jet_nfailures.jl

```diff
@@ -0,0 +1,46 @@
+import Profile
+
+ENV["CI_PERF_SKIP_RUN"] = true # we only need haskey(ENV, "CI_PERF_SKIP_RUN") == true
+
+filename = joinpath(dirname(@__DIR__), "examples", "hybrid", "driver.jl")
+
+# Uncomment for customizing specific jobs / specs:
+# dict = parsed_args_per_job_id(; trigger = "benchmark.jl"); # if job_id uses benchmark.jl
+# dict = parsed_args_per_job_id();                           # if job_id uses driver.jl
+# parsed_args = dict["sphere_aquaplanet_rhoe_equilmoist_allsky"];
+
+try
+    include(filename)
+catch err
+    if err.error !== :exit_profile
+        rethrow(err.error)
+    end
+end
```

Comment:
I'd like a better way to do this....

Reply:
Yeah, we can probably do something cleaner once we further simplify the driver